src/pages/Compare/index.spec.tsx

```diff
@@ -886,6 +916,191 @@ describe(__filename, () => {
     expect(dispatch).not.toHaveBeenCalled();
   });
 
+  describe('preload', () => {
+    const setUpForPreloadAndRender = ({
+      addonId = 2,
+      baseVersionId = 4,
+      buildTree = true,
+      currentFile = 'currentFile.json',
+      headVersionId = 5,
+      loadDiff = true,
+      loadVersionFile = true,
+      nextModifiedFile = 'modifiedFile.json',
+      ...params
+    }) => {
+      const version = createExternalVersionWithEntries(
+        [
+          { path: currentFile, status: 'M' },
+          { path: 'fileWithoutChange', status: '' },
+          { path: nextModifiedFile, status: 'M' },
+        ],
+        { id: headVersionId, selected_file: currentFile },
+      );
+      const {
+        _fetchDiff,
+        _fetchVersionFile,
+        dispatchSpy,
+        fakeThunk,
+      } = loadDiffAndRender({
```

Comment:
There are some defaults here that you don't need to re-define because you're already allowing callers to specifies those values with `...params`. I think it can be simplified to this:

```diff
diff --git a/src/pages/Compare/index.spec.tsx b/src/pages/Compare/index.spec.tsx
index 985f1e7..e229573 100644
--- a/src/pages/Compare/index.spec.tsx
+++ b/src/pages/Compare/index.spec.tsx
@@ -918,13 +918,9 @@ describe(__filename, () => {
 
   describe('preload', () => {
     const setUpForPreloadAndRender = ({
-      addonId = 2,
-      baseVersionId = 4,
       buildTree = true,
       currentFile = 'currentFile.json',
       headVersionId = 5,
-      loadDiff = true,
-      loadVersionFile = true,
       nextModifiedFile = 'modifiedFile.json',
       ...params
     }) => {
@@ -942,12 +938,8 @@ describe(__filename, () => {
         dispatchSpy,
         fakeThunk,
       } = loadDiffAndRender({
-        addonId,
-        baseVersionId,
         buildTree,
         headVersionId,
-        loadDiff,
-        loadVersionFile,
         version,
         ...params,
       });
```